tomb

```diff
@@ -786,7 +786,7 @@ usage() {
 	echo
 	[[ $QRENCODE == 1 ]] && {
 		_print " // Backup on paper:"
-		_print " engrave makes a QR code of a KEY to be saved on paper"
+		_print " engrave      makes a QR code of a KEY to be saved on paper"
```

Comment:
I imagine this is the right way, because all the text has this tabulation. As you can see here when i execute `--help`:

![Captura de tela de 2023-06-26 17-04-41](https://github.com/dyne/Tomb/assets/3987342/0f0c2059-7999-448e-ab2e-83d43caf6d52)

it's in Brazilian Portuguese because i am working on this translation. But you can see the engrave is lined up in my translation.

Reply:
Ah, I see. Missed that, because I don't have that dep installed and didn't notice it was the help text.
Just saw the diff and not the surroundings :D

Reply:
> Or generally asked: Why is the spacing needed?

Ok, is not NEEDED, it's just aesthetics.